gammapy/background/tests/test_fov.py

```diff
@@ -1,2 +1,110 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import print_function, division
+
+from gammapy.background import fill_acceptance_image
```

Comment:
Fixed order of imports.
@cdeil 
What do you mean by relative import? Use:
  from .gammapy.image import make_empty_image
instead of:
  from gammapy.image import make_empty_image
for instance?
Should I use it only for gammapy or also for astropy?